src/serialization/sb3.js

```diff
@@ -811,14 +813,14 @@ const deserializeBlocks = function (blocks) {
             // delete the old entry in object.blocks and replace it w/the
             // deserialized object
             delete blocks[blockId];
-            deserializeInputDesc(block, null, false, blocks);
+            oldToNew[blockId] = deserializeInputDesc(block, null, false, blocks);
             continue;
         }
         block.id = blockId; // add id back to block since it wasn't serialized
         block.inputs = deserializeInputs(block.inputs, blockId, blocks);
         block.fields = deserializeFields(block.fields);
     }
-    return blocks;
+    return ({blocks, oldToNew});
```

Comment:
It seems a bit weird to complicate the signature of this function when the only places that use the old `blocks` return value are the tests.